browser/webpack.config.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+    entry: "./app.js",
+    output: {
+        path: __dirname,
+        filename: "bundle.js"
+    },
+    devtool: 'cheap-module-eval-source-map',
+    node: {
+        fs: "empty"
+    },
+    module: {
+        loaders: [{
+            test: /\.js?$/,
+            loader: 'babel-loader',
+            exclude: /node_modules/,
+            query: {
+                presets: ['es2015', 'stage-0']
+            }
+        }]
```

Comment:
we probably also want a `plugins: [new webpack.optimize.UglifyJsPlugin()]` section here, so that the bundle.js file ends up minified